iothub/device/src/ClientOptions.cs

```diff
@@ -54,5 +54,13 @@ public class ClientOptions
         /// or the <see cref="ModuleClient.CreateFromEnvironmentAsync(ClientOptions)"/> flow.
         /// </remarks>
         public int SasTokenRenewalBuffer { get; set; }
+
+        /// <summary>
+        /// The payload convention to be used to serialize and encode the messages for our convention based methods. The default value is set to <see cref="DefaultPayloadConvention.Instance"/>.
+        /// </summary>
+        /// <remarks>
+        /// The <see cref="PayloadConvention"/> defines both the serializer and encoding to be used for our convetion based messages. You will only need to set this if you have objects that have special serialization rules or require a specific byte encoding.
```

Comment:
I think it would help to call out here that our default is newtonsoft json based and encodes with utf-8

Reply:
Let me add that to see if it makes it more clear.